infra/cifuzz/actions/build_fuzzers/action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
   build-integration-path:
     description: "The path to the the project's build integration."
     required: false
+  do-bad-build-check:
```

Comment:
Why add the "do-" in front, i think bad-build-checks seems better.

Reply:
Done.